src/Dockerfile

```diff
@@ -14,19 +14,20 @@
 #   limitations under the License.
 #*******************************************************************************
 
-FROM python:3-slim
+FROM python:3.9-slim
 
 ENV DEBIAN_FRONTEND noninteractive
 ENV LANG C.UTF-8
 
 # Speculos build dependencies
 RUN apt-get update && \
   apt-get install -qy cmake curl gcc-arm-linux-gnueabihf \
-    git libc6-dev-armhf-cross libvncserver-dev qemu-user-static \
-    tesseract-ocr libtesseract-dev wget && \
+    git libc6-dev-armhf-cross libvncserver-dev python3-pip \
+    qemu-user-static tesseract-ocr libtesseract-dev wget \
```

Comment:
You could remove tesseract I guess.

Reply:
We're not using that now, but it's needed for legacy OCR, isn't it? Being just a py dependency, it seems reasonable to keep it.
Thanks for the review!

Reply:
If you are using a Speculos version >= 0.2.8, Tesseract it not used at all on it.
You can obviously keep it, but it is unused and just increase the image size for no reason.